doc/buildAndProgram.md

```diff
@@ -75,7 +75,7 @@ If you just want to build the project and run it on the Pinetime, using *pinetim
 Build:
 
 ```
-make -j pinetime-app
+make -j $(nproc) pinetime-app
```

Comment:
I'm debating between removing this, keeping this or writing a better explanation. Setting the number of jobs isn't necessary for the build, and this unexplained option could confuse newbies, however it should probably always be set anyway. We could add a comment below about using the `-j` option to speed up the build, but I'm not sure it's our docs job to explain.

Reply:
While it doesn't seem like it should be our job to explain what the `-j` option means, I think it would be more confusing to have a build run really slowly, than entering an unknown option. I propose either adding a very small note about that the `-j` option makes it use more threads, just have an unexplained option, or even switch to Ninja, which uses all threads by default.

Reply:
I think we should set it to 4 for now. This is better than nothing, less intimidating and easier to type out multiple times.